lib/config.js

```diff
@@ -1,44 +1,51 @@
-var defaults = require('lodash.defaults');
-var compact = require('lodash.compact');
-var map = require('lodash.map');
-var fs        = require('fs');
-var path      = require('path');
-var osHomedir = require('os-homedir');
+'use strict';
 
-var config = null;
+const defaults = require('lodash.defaults');
```

Comment:
To clarify, I meant code style cleanup including alignment of operators like this. It's pretty common to manually align assignments, function parameters, etc., for readability.

Responding more directly to "Why would I be opposed?", the question was due to this exchange:

> -- We'll eventually want to align this equals sign with the ones below, right?
> -- Not unless that is actually forced by the linter.

I read that as "we will not want to align this equals sign with the ones below", which I found odd (hence my question), but "I would actually unalign them" confirms that I read you correctly. In any case, this is a rather minor issue so I wouldn't insist on any particular format. Carry on as you prefer :)